.travis.yml

```diff
@@ -35,8 +35,12 @@ matrix:
       env: RAILS_VERSION=5.1 SEQUEL_VERSION=5.0
     - rvm: 2.6.0
       env: RAILS_VERSION=5.2 SEQUEL_VERSION=5.0
+    - rvm: 2.6.0
```

Comment:
Rails 6 [works from Ruby version `2.5.0` or up](https://guides.rubyonrails.org/getting_started.html#installing-ruby), we should add that to the matrix as well.